services/httpd/handler.go

```diff
@@ -954,14 +955,17 @@ func parseSystemDiagnostics(d *diagnostics.Diagnostics) (map[string]interface{},
 }
 
 // httpError writes an error to the client in a standard format.
-func (h *Handler) httpError(w http.ResponseWriter, error string, code int) {
```

Comment:
I'll settle on `errmsg`. I don't like `error` as a variable name as it's also a predeclared identifier and my editor paints it all bright purple. 😄